subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/PerformanceScenario.java

```diff
@@ -36,6 +36,10 @@ public String getClassName() {
         return className;
     }
 
+    public String getSimpleClassName() {
+        return className.substring(className.lastIndexOf('.') + 1);
+    }
```

Comment:
Let's see how this works out, but I think we should just make this part of the display name, and use it everywhere.

Reply:
I added it now to the display name. I'll run a regression performance test suite, to see what the full report looks like. In my small test it doesn't look to bad: 
![image](https://user-images.githubusercontent.com/423186/95322277-88531a00-089c-11eb-90cd-1c3bd8a17bec.png)